files/system/usr/share/ublue-os/just/70-secureblue.just.readme.md

```diff
@@ -78,6 +78,12 @@
 
 `gather_data_sampling=force`
 
+### Force disable simultaneous multithreading
+
+**Disables this hardware feature on user request, regardless of mitigation necessity**
```

Comment:
maybe "regardless of whether the hardware is affected by a known vulnerability that would require a mitigation"?

I'm inclined to err on the side of verbosity

Reply:
Updated according to your suggestion, with "vulnerabilities" in plural (nit).